### PR TITLE
[Standalone] Bundle binaries for clipboard access in Windows/Linux

### LINKED ISSUE #21: Fetch clipboard from other side (files) fails
When I try to `fetch clipboard from other side` (which is actually on the same host running Debian 10)
I get an `copy clipboard failed` error.

**The content of the `client-clipboard-1609069044630.txt`**:
```
x-special/nautilus-clipboard
copy
file:///home/foo/bar/test/dvi%20(Kopie).jpg
file:///home/foo/bar/test/Foo
```

**The error thrown in the terminal**:
```
Error: Couldn't find the 'xsel' binary and fallback didn't work. On Debian/Ubuntu you can install xsel with: sudo apt install xsel
    at makeError (/snapshot/ezshare/node_modules/clipboardy/lib/linux.js:14:11)
    at xselWithFallback (/snapshot/ezshare/node_modules/clipboardy/lib/linux.js:31:10)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

### PR DESCRIPTION
**Pull request by Maxime-J**
Fix #21

Windows version will use the 64-bit binary (because current github actions config is 64-bit)
Linux version will use the bundled xsel (fallback leaved and set to system)

### CODE REVIEW

File: standalone.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
++const xsel = path.join(path.dirname(process.execPath), 'clipboard');
++const xselFallback = 'xsel';
+ 
+ const copyArguments = ['--clipboard', '--input'];
+ const pasteArguments = ['--clipboard', '--output'];
+ 
+ const makeError = (xselError, fallbackError) => {
+ 	let error;
+-	if (xselError.code === 'ENOENT') {
++	if (fallbackError.code === 'ENOENT') {
```
Comment (mifi) [inline standalone.patch:18]: Why this change?

**Review by mifi — COMMENTED** [review]
Thanks for your PR! I have left a question

File: standalone.patch (hunk 1 above)
Comment (Maxime-J) [reply standalone.patch:18]: Clipboardy by default first tries the system xsel and then fallback to its shipped xsel if the system one is not found.
As we bundled it here, I thought we might as well use it by default.

So I switched and set the fallback to the system one (even if there's probably no reason for it to happen)
That change is to make the error check right.

Comment (mifi) [reply standalone.patch:18]: Ohh I see. I'm thinking maybe it's better to use the system one first? Because the fallback one may be for the wrong CPU architecture etc, and I'm thinking it's nice to use the system one if that works (calling a downloaded executable could cause problems like blocking popup dialogs prompting for user to confirm execution etc)

**Comment by mifi** [comment]
Thanks!
